acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -5,6 +5,22 @@
 
 test_name "Install Packages"
 
+def func_use_system_openssl?
+  # Presently .i.e as of Oct 22 there is no way to specify an environment variable
+  # in PA CI so we are going to hard code it till there is support for FIPS platforms
+  return true
```

Comment:
This doesn't make any sense to me... we don't want to use system openssl in all our tests. Also, it should be easy to add an environment variable in PA CI...